index.js

```diff
@@ -0,0 +1,73 @@
+const testLineup = ['1B', '2B', '3B', 'C', 'OF', 'OF', 'OF', 'P', 'SS']
+
+const validateLineup = (lineup) => {
+  return (checkSalary(lineup) && checkPlayerTeams(lineup) && checkGames(lineup) && checkPositions(lineup, testLineup))
+}
+
+// Check Salary
+const checkSalary = (lineup) => {
+  let totalSalary = 0
+
+  lineup.forEach(player => totalSalary += player.salary)
+
+  return (totalSalary <= 45000) ? true : false
```

Comment:
use reduce function
const initialSalary = 0
return lineup.reduce((previousSalary, player) => previousSalary + player.salary, initialSalary) <= 45000

Reply:
if you need help understanding it. let me know